library/src/main/scala/scala/scalajs/js/typedarray/TypedArray.scala

```diff
@@ -65,6 +65,21 @@ trait TypedArray[T, Repr] extends ArrayBufferView with js.Iterable[T] {
  *  Static information that exists for any concrete TypedArray
  */
 @js.native
-trait TypedArrayStatic extends js.Object {
+trait TypedArrayStatic[T, Repr] extends js.Object {
   val BYTES_PER_ELEMENT: Int = js.native
+
+  /** Returns a new array from a set of elements. */
+  def of(items: T*): Repr = js.native
+
+  /** Creates an array from an `arrayLike` or iterable object. */
+  def from(arrayLike: js.Array[T]): Repr = js.native
+
+  /** Creates an array from an `arrayLike` or iterable object. */
+  def from[E](arrayLike: js.Array[E], mapfn: js.Function1[E, T]): Repr = js.native
```

Comment:
[Like TypeScript defines so](https://github.com/microsoft/TypeScript/blob/b36c8a06902ca47da1568167991bd0bc28ced80e/src/lib/es5.d.ts#L1396-L1399), I think `arrayLike` should be a strcture type like
```scala
def from(arrayLike: { 
    def length: Int
    @JSBracketAccess def get(n: Int): T
})
```

But I am not sure what is a Scala.js equilvalent for that.

Reply:
There is no equivalent in Scala.js. Structural JS types are not supported. Just ignore the array-like's that are not true arrays nor true iterables. If someone really wants to call `from` with a non-iterable array-like, they can cast it to a `js.Array`.